.travis.yml

```diff
@@ -139,6 +139,10 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
         singlenode-cassandra -g cassandra
     fi
+    if [[ -v PRODUCT_TESTS_SPECIFIC_ENVIRONMENT ]]; then
+      presto-product-tests/bin/run_on_docker.sh \
+        multinode-tls-kerberos -g smoke,cli,group-by,join,tls -x stats_client
```

Comment:
Personally i would just drop the `smoke` group here, so you don't have do exclude the `stats_client` group.

Reply:
The idea with `group-by` and `join` here is to make the Presto do the exchanges.